google/cloud/apigee/registry/applications/v1alpha1/diff_analytics.proto

```diff
@@ -1,5 +1,5 @@
 syntax = "proto3";
-package google.cloud.apigee.registry.applications.v1;
+package google.cloud.apigee.registry.v1alpha1;
```

Comment:
Let's make this package name "google.cloud.apigee.registry.applications.v1alpha1"